eng/scripts/get-aspire-cli.ps1

```diff
@@ -0,0 +1,437 @@
+#!/usr/bin/env pwsh
+
+[CmdletBinding()]
+param(
+    [string]$OutputPath = "",
+    [string]$Channel = "9.0",
```

Comment:
The default being `9.0` is weird becuase there was no CLI in the 9.0 version. If we want `9.0` to mean `9.x`, I think the channel should be named `9` or `9.x`.

I also wonder if we shouldn't have the concept of a "channel" but instead just use versions - `9` or `9.3`, `9.3.1`, etc. See also https://github.com/dotnet/aspire/issues/9642#issuecomment-3014385006

Reply:
I can rename the parameter to `Version`.

Reply:
With the idea that I can specify a sub-version? like `9` or `9.3` or `9.3.1`?

Reply:
For the script it can just be the Version string, and in the builds we can map these to the builds appropriately. Does that sound good?

Reply:
done.